api/src/models/request_info.rs

```diff
@@ -57,6 +58,11 @@ impl RequestInfo {
         self.base_url.scheme()
     }
 
+    // Returns the forwarded port.
+    pub fn port(&self) -> &str {
```

Comment:
I think `port`, `scheme`, and `host` are methods that aren't used anywhere (I think I see some compiler warnings all the time). Therefore, I would like to remove them.